scripts/toolchain/install-toolchain.sh

```diff
@@ -170,6 +170,32 @@ function install_wasm_opt() {
     fi
 }
 
+# used by rust libp2p to compile objects
+function install_protobuf() {
+  local protobuf_version=21.12
+  echo "Installing protobuf version 3.${protobuf_version}"
+  local ostype="$(uname -s)"
+  local cputype="$(uname -m)"
+  case "${ostype}" in
+      Linux | linux)
+          ostype="linux"
+          ;;
+      Darwin)
+          ostype="macos"
+          ;;
+      *)
+          echo "no precompiled binaries available for OS: ${ostype}"
+      ;;
+  esac
+  PB_REL="https://github.com/protocolbuffers/protobuf/releases"
+  curl -fsSLO "${PB_REL}/download/v21.12/protoc-21.12-${ostype}-${cputype}.zip"
```

Comment:
```suggestion
  curl -fsSLO "${PB_REL}/download/v${protobuf_version}/protoc-${protobuf_version}-${ostype}-${cputype}.zip"
```

Reply:
Updated to fetch the protoc to /tmp/ and unzip from there. No longer should the file be problematic.